### ISSUE TITLE
Still planning active development?

### ISSUE THREAD

**Issue by WilfredTA** [KeenS/webml#90]
Title says it all; just wondering if there are still plans to complete this?

**Comment by KeenS** [comment]
Yes. I'm planning to develop to conform SML'97 and other unique features like web integration.  Just development is too slow.

**Comment by WilfredTA** [comment]
Awesome to hear. Let me know if there are any current issues that you particularly welcome external contributions. If there are meta-level tasks, happy to talk about that as well!
